### PR TITLE
More FAQ

### PR DESCRIPTION
**Pull request by kba**
Additions welcome. Answers too :)

### CODE REVIEW

File: faq.md [hunk 1]
```diff
@@ -1,6 +1,30 @@
...
+
+### Who is the target audience of OCR-D?
+
+### Where can I get support on OCR-D?
+
+### What is the difference between OCR-D and ABBYY?
+
+### What is the difference between OCR-D and Tesseract?
+
+### What is the difference between OCR-D and TRANSKIBUS?
```
Comment (EEngl52) [inline faq.md]: replace "TRANSKIBUS" with "TRANSKRIBUS"

File: faq.md [hunk 2]
```diff
@@ -10,6 +34,44 @@ ocrd workspace --help
...
+
+## Workflows and processors
+
+### How can I define workflows?
+
+### Where can I find sample workflows to experiment with?
+
+### How to handle failing workflows?
+
+### Why do some processors have multiple input or output file groups?
```
Comment (EEngl52) [inline faq.md:184]: Also: "How do I define them"

Comment (kba) [reply faq.md:184]: You mean like `How do I specify multiple input/output file groups?`

File: faq.md [hunk 3]
```diff
@@ -1,6 +1,30 @@
...
+### What is the difference between OCR-D and Tesseract?
+
+### What is the difference between OCR-D and TRANSKIBUS?
+
+### What is the difference between OCR-D and ocrd4all?
+
+### Is OCR-D production-ready?
+
+### How much does it cost to deploy OCR-D?
+
```
Comment (EEngl52) [inline faq.md:84]: What are the system requirements for OCR-D-software?

File: faq.md [hunk 4]
```diff
@@ -1,6 +1,30 @@
...
+### What is the difference between OCR-D and ABBYY?
+
+### What is the difference between OCR-D and Tesseract?
+
+### What is the difference between OCR-D and TRANSKIBUS?
+
+### What is the difference between OCR-D and ocrd4all?
+
+### Is OCR-D production-ready?
+
```
Comment (EEngl52) [inline faq.md:60]: Which formats are supported by OCR-D?

File: faq.md (hunk 2 above)
Comment (EEngl52) [reply faq.md:184]: Yes

File: faq.md [hunk 5]
```diff
@@ -10,6 +34,44 @@ ocrd workspace --help
...
+### Which processors are available?
+
+## Workflows and processors
+
+### How can I define workflows?
+
+### Where can I find sample workflows to experiment with?
+
+### How to handle failing workflows?
+
```
Comment (EEngl52) [inline faq.md:183]: Why does OCR-D need METS files? How can I process images without METS?

Comment (wrznr) [reply faq.md:183]: Not possible, not desired. The METS file is actually not needed but rather provided by the framework.

File: setup.md [hunk 6]
```diff
@@ -85,6 +85,8 @@ docker pull ocrd/all:maximum
 
 Replace `maximum` accordingly if you want the `minimum` or `medium` variant.
 
+If you do not specify a version, it chooses `latest` which is equivalent to `medium`.
```
Comment (cneud) [inline setup.md]: If no specific version is chosen, `latest` is selected by default, which is equivalent to `medium`.

File: faq.md (hunk 5 above)
Comment (kba) [reply faq.md:183]: Feel free to add directly to the PR, I will clean up before merge.

**Review by cneud — APPROVED** [review]
Great!

**Comment by kba** [comment]
Fixing locale-related problems

**Review by EEngl52 — APPROVED** [review]

**Comment by kba** [comment]
Merged in https://github.com/OCR-D/ocrd-website

**Pull request closed without merging**